app/javascript/components/MainContainer/MainContainer.js

```diff
@@ -0,0 +1,86 @@
+import React, { Component } from 'react';
+import RestaurantsList from "./RestaurantsList/RestaurantsList";
+import Filters from "./Filters/Filters"
+import Map from "./Map/Map"
+import './main_container.css'
+const axios = require('axios');
+
+
+class MainContainer extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            rating_filter: 'all',
+            cuisine_filter: 'all',
+            ten_bis_filter: 'all',
+            search_text: '',
+            restaurants: [],
+            filtered_restaurants: [],
+            cuisines: []
+        };
+
+        this.onChangeFilter = this.onChangeFilter.bind(this);
+        this.onChangeSearch = this.onChangeSearch.bind(this);
+        this.filterRestaurants = this.filterRestaurants.bind(this);
+
+    }
+
+    componentDidMount(){
+        axios.get('/restaurants.json')
+            .then((response) => {return response.data;})
+            .then((data) => {
+                let cuisines = [...new Set(data.map(x=>x.cuisine.toLowerCase()))].sort();
+                cuisines.unshift('all');
+                this.setState({ restaurants: data, filtered_restaurants: data, cuisines: cuisines });
```

Comment:
If you have an object with a key and value of the same name, you can write it as short hand:
```
const foo = whaterver;
{ foo } == { foo: foo }
```